src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -216,197 +198,273 @@
+
+                                            <!--  Temporary badge replacement for InfoBadge control (htcfreek). (Normally you need one grid per icon group. But if you want to show two badges at the same place in a StackPanel you have to put all FontIcons into the same Grid.)  -->
+                                            <Grid>
+                                                <!--  Error badge  -->
+                                                <FontIcon
+                                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                                    Foreground="{ThemeResource InfoBarErrorSeverityIconBackground}"
+                                                    Glyph="{StaticResource InfoBarIconBackgroundGlyph}"
+                                                    Visibility="{x:Bind ShowBadgeOnPluginSettingError}" />
+                                                <FontIcon
+                                                    FontFamily="{StaticResource SymbolThemeFontFamily}"
+                                                    Foreground="{ThemeResource InfoBarErrorSeverityIconForeground}"
+                                                    Glyph="{StaticResource InfoBarErrorIconGlyph}"
+                                                    Visibility="{x:Bind ShowBadgeOnPluginSettingError}" />
+                                            </Grid>
+
+                                            <ToggleSwitch
+                                                x:Uid="PowerLauncher_EnablePluginToggle"
+                                                IsOn="{x:Bind Path=Disabled, Converter={StaticResource BoolNegationConverter}, Mode=TwoWay}" />
+                                        </StackPanel>
+                                        <labs:SettingsExpander.Items>
+                                            <labs:SettingsCard
+                                                x:Uid="PowerLauncher_ActionKeyword"
+                                                IsEnabled="{x:Bind Enabled, Mode=OneWay}">
+                                                <TextBox
+                                                    MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                                    Text="{x:Bind Path=ActionKeyword, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+                                            </labs:SettingsCard>
+                                            <labs:SettingsCard
+                                                Padding="0,-4,0,0"
+                                                HorizontalContentAlignment="Left"
+                                                Background="{ThemeResource SystemFillColorCriticalBackgroundBrush}"
+                                                ContentAlignment="Vertical"
+                                                Visibility="{x:Bind ShowNotAccessibleWarning}">
```

Comment:
Is this value a boolean or not? 'cause here it should be a Visibility and in line 360 it's a boolean.